__tests__/cli/list/directory-contents/DirectoryContents.handler.test.ts

```diff
@@ -83,4 +82,49 @@ describe("directory-contents Handler", () => {
 
         expect(Files.listDirectoryContents).toHaveBeenCalledWith(fakeDir);
     });
+
+    it("should read a mocked directory, even if checkStatus fails", async () => {
+        const fakeDir: string = "fake/directory";
+        // Mock the files api call
+        Files.listDirectoryContents = jest.fn((dir) => {
+            return [{
+                fake: "file",
+                moreFake: "directory"
+            }];
+        });
+        CheckStatus.getZosmfInfo = jest.fn(async () => {
+            throw "dummy error";
+        });
+        ZosmfSession.createBasicZosmfSession = jest.fn();
+        const handler = new DirectoryHandler.default();
+        const params = Object.assign({}, ...[DEFAULT_PARAMTERS]);
```

Comment:
Should we fix the typo `DEFAULT_PARAMTERS` -> `DEFAULT_PARAMETERS` across all tests?